proposed/cache-meta.md

```diff
@@ -279,18 +279,20 @@ do so.
 5. People
 ---------
 
-### 5.1 Editor
+### 5.1 Coordinator
+* Paul Dragoonis
+
+### 5.2 Editor
 
 * Larry Garfield
 
-### 5.2 Sponsors
+### 5.3 Sponsors
 
-* Beau Simensen, Sculpin (Coordinator)
+* Beau Simensen, Sculpin
 * Pádraic Brady, Zend Framework
```

Comment:
@Crell so I'd be cool with me under Contributor section with (coordinator). I've never been a sponsor.